### ISSUE TITLE
app crash after WIN11 22H2

### ISSUE THREAD

**Issue by devgijbels** [ikvm-revived/ikvm#179]
Debugging shows the crash is caused by ikvm function java.lang.System.setProperty. Any help would be greatly appreciated.

event log entry:
-------------------------------------------------------------
Naam van toepassing met fout: xxxxxxxx.exe, versie: 6.14.2.0, tijdstempel: 0x63317922 

Naam van module met fout: clr.dll, versie: 4.8.9075.0, tijdstempel: 0x62cce3b7 

Uitzonderingscode: 0xc00000fd 

Foutmarge: 0x004592ea 

Id van proces met fout: 0x0x25F4 

Starttijd van toepassing met fout: 0x0x1D8D8968BBCB7D7 

Pad naar toepassing met fout: C:\xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.exe 

Pad naar module met fout: C:\Windows\Microsoft.NET\Framework\v4.0.30319\clr.dll 

Rapport-id: 23a99297-1b6b-4562-bb7d-9aa0ae9b8936 

Volledige pakketnaam met fout: 

Relatieve toepassings-id van pakket met fout:

**Comment by NightOwl888** [comment]
We don't exactly provide support version 7.x of IKVM (we are a new team that has only released 8.2.0 and up). Is there any chance you can use the latest version?

Also, it looks like you are not using an official version...? There wasn't an official release for 7.4.5256: https://www.nuget.org/packages/IKVM

**Comment by wasabii** [comment]
Yeah, we are not the maintainers of the IKVM.OpenJDK.* set of packages. Those were uploaded almost 10 years ago, by somebody not even affiliated with the original IKVM developers, as far as I know.

**Comment by wasabii** [comment]
I'm going to go ahead and close this until a bug in this project is identified. Feel free to open a new issue.
